eth-contracts/package.json

```diff
@@ -5,56 +5,52 @@
   "author": "AudiusProject",
   "license": "Apache-2.0",
   "scripts": {
-    "ganache": "docker run --name audius_ganache_cli_eth_contracts -d -p 8546:8545 trufflesuite/ganache-cli:v6.9.1 -h 0.0.0.0 -a 50 -l 8000000",
-    "ganache-test": "docker run --name audius_ganache_cli_eth_contracts_test -d -p 8556:8545 trufflesuite/ganache-cli:v6.9.1 -h 0.0.0.0 -a 50 -l 8000000",
-    "ganache-i": "docker ps",
-    "ganache-q": "docker rm -f audius_ganache_cli_eth_contracts > /dev/null",
-    "ganache-test-q": "docker rm -f audius_ganache_cli_eth_contracts_test > /dev/null",
     "postinstall": "patch-package",
-    "truffle-compile": "./node_modules/.bin/truffle compile",
-    "truffle-migrate": "./node_modules/.bin/truffle migrate --reset --network development",
-    "truffle-migrate-test": "./node_modules/.bin/truffle migrate --reset --network test_local",
-    "truffle-console": "./node_modules/.bin/truffle console --network development",
-    "truffle-console-test": "./node_modules/.bin/truffle console --network test_local",
-    "test": "npm run truffle-test",
-    "test-coverage": "ENABLE_OPTIMIZER=false node --max-old-space-size=4096 ./node_modules/.bin/truffle run coverage --file='test/*.js'",
-    "test-only": "./node_modules/.bin/truffle test test/*.js --network=test_local",
-    "test-only-dev": "./node_modules/.bin/truffle test test/*.js --network=development",
-    "test-random": "./scripts/truffle-test.sh --audius-random",
-    "truffle-test": "./scripts/truffle-test.sh",
-    "truffle-test-v": "./scripts/truffle-test.sh --verbose-rpc",
-    "test-circle-ci": "./scripts/circleci-test.sh",
-    "test-circle-ci-no-coverage": "./scripts/circleci-test-no-coverage.sh",
-    "solidity-lint": "./node_modules/.bin/solium -d contracts",
-    "lint": "./node_modules/.bin/standard",
-    "lint-circle-ci": "./scripts/lint.sh"
+    "truffle:compile": "truffle compile",
+    "truffle:migrate": "truffle migrate --reset --network development",
+    "truffle:console": "truffle console --network development",
+    "test": "truffle test test/*.test.js --network test",
+    "test:random": "truffle test test/random/random.test.js --network development",
+    "test:verbose": "truffle test test/*.test.js --network development --verbose-rpc",
+    "coverage": "ENABLE_OPTIMIZER=false node --max-old-space-size=4096 truffle run coverage --file='test/*.js'",
+    "solidity:lint": "solium -d contracts",
+    "js:lint": "eslint .",
+    "lint": "npm run js:lint && npm run solidity:lint",
+    "analysis": "slither ."
   },
   "dependencies": {
-    "@aragon/court": "^1.1.2",
+    "@aragon/court": "1.1.3",
     "@openzeppelin/contracts-ethereum-package": "2.5.0",
     "@openzeppelin/upgrades": "2.8.0",
-    "@truffle/hdwallet-provider": "^1.0.13",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "ethereumjs-abi": "0.6.7",
+    "@truffle/hdwallet-provider": "1.7.0",
```

Comment:
Just checking you meant to lock all of these to exact versions?

Reply:
Yep that's right. The dependencies are kind of fragile, and also to prevent supply chain style attacks